chromium_src/chrome/browser/ui/browser.cc

```diff
@@ -52,6 +52,10 @@ Browser::CreateParams::CreateParams(Type type, Profile* profile)
 
 Browser::CreateParams::CreateParams(const CreateParams& other) = default;
 
+content::WebContents* Browser::OpenURL(const content::OpenURLParams& params) {
+  return NULL;
```

Comment:
I'm a little worried about this one causing a crash. Where does it get called from? Is it just windows doing its usual terrible job of dead code elimination or is there an actual legit code path that leads here?